ingestion/tests/unit/topology/database/test_deltalake.py

```diff
@@ -223,4 +226,7 @@ def test_yield_table(self):
             schemaDefinition=None,
         )
 
+        print(table_request)
```

Comment:
i guess this was for development. I suggest removing as print statements in succeeding tests can be confusing.

Reply:
Thanks for catching that